en/docs/reference/connectors/salesforce-connectors/sf-inbound-endpoint-reference-configuration.md

```diff
@@ -73,13 +73,28 @@ The following configurations allow you to configure Salesforce Inbound Endpoint
     <td><code>5000</code></td>
     <td><code>5000</code></td>
   </tr>
+  <tr>
+    <td><code>connection.salesforce.initialEventId</code></td>
+    <td>The initial event id to start message reading. If message replaying is enabled, this value will be used only if <code>connection.salesforce.EventIDStoredFilePath</code> is not configured and if this <code>initialEventId</code> is greater than the replayId stored in the registry.
```

Comment:
```suggestion
    <td>Specifies the initial event ID from which to start reading messages. This value is applicable only when message replaying is enabled. It will be used only if <code>connection.salesforce.EventIDStoredFilePath</code> is not configured, and the value of this parameter(<code>initialEventId</code>) is greater than the replay ID stored in the registry.
```